src/main/java/ca/corefacility/bioinformatics/irida/ria/web/models/tables/TableRequest.java

```diff
@@ -40,11 +40,11 @@ public void setSortColumn(String sortColumn) {
 	}
 
 	public String getSearch() {
-		return search.trim();
+		return search;
 	}
 
 	public void setSearch(String search) {
-		this.search = search;
+		this.search = search.trim();
```

Comment:
Wouldn't this still through an error if the search argument is `null`?

Reply:
I don't think it should happen that a null ever gets set here, but I did it just in case in 90b5c05.

Reply:
Thanks, that looks good.  Agreed thought that it shouldn't but safe is good.